addons/better-img-uploads/userscript.js

```diff
@@ -150,7 +150,7 @@ export default async function ({ addon, console, msg }) {
       } //Otherwise just leave the image the same size
 
       function getResizedWidthHeight(oldWidth, oldHeight) {
-        const STAGE_WIDTH = 479;
+        const STAGE_WIDTH = 480;
```

Comment:
This was initially changed to 479 by commit https://github.com/ErrorGamer2000/ScratchAddons/commit/6da9ddc95ba8d999014dfe781b7bdbbd43dc480f#diff-854cc3c818d6760de96084d1f214859d4919296db14211aa9204b9c5b218cd4cR114

It was probably unintentional, it's based on vanilla Scratch code which uses 480 of course: https://github.com/scratchfoundation/scratch-svg-renderer/blob/a4ad7080761a9dba8c5ffadad703cfd07f62841a/src/bitmap-adapter.js#L70